### DISCUSSION TITLE
Question about asynchronous projection

### DISCUSSION THREAD

**Discussion by liujingshi** [JasperFx/marten#2499] [General]
When my event write completes, the asynchronous projection was be running, but at the same time my system dose a query projection. My problem is that the asynchronous projection was not  written completes when I query, 
So I did not always get the data when I want.  so Could you tell me What is the best solution to this problem?

**Comment by jeremydmiller** [comment]
So it's an issue with eventual consistency getting you here? And I'm converting this to a conversation. Or you may get better results quicker by going to Discord --> https://discord.gg/ppv6ZeKc
